examples/bert/glue.py

```diff
@@ -123,6 +123,8 @@ def create_job_configs(main_config: om.DictConfig, tasks_to_run: Set[str],
                        pretrained_checkpoint_path: Optional[str]):
     configs = []
     for task_name, task_config in main_config.tasks.items():
+        main_config.base_run_name = main_config.get(
+            'base_run_name') or os.environ.get('COMPOSER_RUN_NAME', 'glue')
```

Comment:
```suggestion
        if main_config.get('base_run_name') is None:
            main_config.base_run_name = os.environ.get('COMPOSER_RUN_NAME', 'glue')
```
since you're using 2 lines anyway.

Reply:
Agreed. This "or" business in python is subtle. Might as well stay away from it if it doesn't save you lines.